peep.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get pip version number
+PIPVER=`pip --version | awk '{print $2}'`
+ARGS=""
+
+echo "peep.sh: Using pip $PIPVER"
+
+# If we're using pip 1.5, 1.6 or 6.0, then toss on the --no-use-wheel
+# argument.
+#
+# Note: If we encounter other versions of pip here that require other
+# things, we can toss them in.
+case $PIPVER in
+    1.5*|1.6*|6.0*)
+        echo "peep.sh: Wheel-using pip detected, so passing --no-use-wheel."
+        ARGS="--no-use-wheel"
+        ;;
+    6.1*)
```

Comment:
I suspect that future versions of pip might start working, actually, due to pypa/pip#2655. Either way, I plan on watching to see what happens with pip and peep.